elements_imaging/imaging.py

```diff
@@ -340,7 +340,7 @@ def make(self, key):
 
             # ---- iterate through all s2p plane outputs ----
             rigid_mc, nonrigid_mc, nonrigid_blocks = {}, {}, {}
-            summary_imgs = []
+            summary_images = []
```

Comment:
could you please expand the other abbreviations too: `rigid_mc`, `mc_key`, `align_chn`, `ops`.
Also, keep the lines to a reasonable length; I think 95 characters was the length we agreed on.